nucypher/policy/conditions/lingo.py

```diff
@@ -116,6 +118,19 @@ def eval(self, data) -> bool:
                 f"'{self.value}' is an unprocessed context variable and is not valid "
                 f"for condition evaluation."
             )
+        if is_context_variable(self.key):
+            # programming error if we get here
+            raise RuntimeError(
+                f"'{self.value}' is an unprocessed context variable and is not valid "
```

Comment:
`self.value` -> `self.key` (I know this PR is still WIP, and still needs preprocessing of key's context variables)

Reply:
eugh, stupid copy/paste

I think the preprocessing will be the same as for `value` - I couldn't come up with any other scenarios